### DISCUSSION TITLE
How to define parameter variability？

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#832] [Questions & Problems] [answered]
Dear to all,
While running a simulation for a population I wanted to incorporate some variability (as SD) for some important parameters ,i.e. renal clearance rate,GFR specific,gastric emptying time and so on,how to do it?
Kind regards！
![image](https://user-images.githubusercontent.com/63133525/90258274-e3f3bf00-de7a-11ea-92ea-6b8777b76e4c.png)

**Comment by tobiasK2001** [comment, marked as answer]
Dear Wenjing,

I am not sure what you are up to but from your created population there is already variability included for renal CL/ GFR and gastric emptying . E.g. via the GFR specific here:
![image](https://user-images.githubusercontent.com/25585248/90608003-4d205d00-e202-11ea-9f5c-e7d03a1c2a25.png)

and in the  GIT transit time e.g. here:
![image](https://user-images.githubusercontent.com/25585248/90608118-6cb78580-e202-11ea-9d19-843384394a8d.png)

I am not sure if it is prudent to add more variability in this parameters. 
Hope this helps.

Best, Tobias

**Comment by Wenjing1018** [comment]
@tobiasK2001 ，
Your answer has solved my doubts，Sincere greetings！
